### PR TITLE
feat(cli): add `envd images remove` command

### LINKED ISSUE #975: feat(CLI): Add envd images remove command
## Description

We lack the command `envd images remove` to remove the envd images. The environment can be removed via `envd destroy`, but the built images cannot be removed using `envd`. This issue is to support it.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by Triple-Z**
resolves #975

### CODE REVIEW

File: pkg/app/image_remove.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	Name:    "remove",
+	Aliases: []string{"r"},
+	Usage:   "Remove an envd image",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "image",
+			Usage:   "Specify the image name to be removed",
+			Aliases: []string{"i"},
+		},
+		&cli.StringFlag{
```
Comment (gaocegege) [inline pkg/app/image_remove.go:37]: Could we support --image image-name:tag?

Comment (Triple-Z) [reply pkg/app/image_remove.go:37]: So the `images remove` command should only have one `--image` flag, and no need for `--tag` flag.

The possible usage like the following.
```
envd images remove --image name  # with default tag `dev`
envd images remove --image name:tag
```

Comment (gaocegege) [reply pkg/app/image_remove.go:37]: The default tag should be `latest`, I think.

Comment (gaocegege) [reply pkg/app/image_remove.go:37]:
```
envd images remove --image name  # with default tag `latest`
envd images remove --image name:tag
```

Comment (Triple-Z) [reply pkg/app/image_remove.go:37]: But the default tag in `envd build` is `dev`. 

https://github.com/tensorchord/envd/blob/f26fc4ce7e42ed0e567cb492c41551f6c2f2c3b4/pkg/app/build.go#L197-L200

Comment (Triple-Z) [reply pkg/app/image_remove.go:37]: > Could we support --image image-name:tag?

Supported, PTAL @gaocegege

**Review by gaocegege — APPROVED** [review]
/lgtm

**Pull request merged**